src-cpp/server42/generate_java_protobuf.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+if [ ! -f "src-java" ]; then
```

Comment:
I think it should be "-d" instead of "-f".

Reply:
good catch